[2025-02-21T07:25:50.395+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T00:02:00+00:00 [queued]>
[2025-02-21T07:25:50.448+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T00:02:00+00:00 [queued]>
[2025-02-21T07:25:50.449+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-02-21T07:25:50.532+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_task> on 2025-01-01 00:02:00+00:00
[2025-02-21T07:25:50.615+0000] {standard_task_runner.py:60} INFO - Started process 1057 to run task
[2025-02-21T07:25:50.622+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'optum_etl_dag1', 'load_task', 'scheduled__2025-01-01T00:02:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/optum_etl_dag1.py', '--cfg-path', '/tmp/tmp9lkvpwbl']
[2025-02-21T07:25:50.662+0000] {standard_task_runner.py:88} INFO - Job 54: Subtask load_task
[2025-02-21T07:25:51.032+0000] {task_command.py:423} INFO - Running <TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T00:02:00+00:00 [running]> on host 491940227afd
[2025-02-21T07:25:51.692+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='optum_etl_dag1' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T00:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T00:02:00+00:00'
[2025-02-21T07:25:51.761+0000] {logging_mixin.py:188} INFO - Loaded data: {'name': 'AIRFLOW', 'type': 'ORCHESTRATION'}
[2025-02-21T07:25:51.763+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-02-21T07:25:51.880+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=optum_etl_dag1, task_id=load_task, execution_date=20250101T000200, start_date=20250221T072550, end_date=20250221T072551
[2025-02-21T07:25:52.238+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-21T07:25:52.315+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
